docs/wiki/archive/Magnetometer.md

```diff
@@ -4,31 +4,37 @@ import MagOrientation from '/img/MagOrientationDiagram.png'
 
 :::warning
 
-Do **NOT** enable a magnetometer unless you have confirmed that:
+Do **NOT** use a magnetometer unless you have confirmed that:
 
 - the calibration is accurate
 - its output is correctly oriented
 - the returned values are clean and noise-free
+- the correct local declination angle is entered in the CLI
 - the heading returned by the mag is correct
 - Acc is enabled, correctly oriented, and calibrated
 - GPS is included in the firmware and enabled, to see the heading data
-  :::
+
+:::
 
 ## Introduction
 
-The purpose of a magnetometer is to return the 'heading' of the quad, meaning the direction the nose of the quad is pointing in. More precisely, we want the angle in degrees between the nose of the quad and North. When set up correctly, the returned value should, for example, be 90 degrees when the nose of the quad is pointing due East, 180 degrees when pointing due south, etc.
+The purpose of a magnetometer is to return the 'heading' of the quad, meaning the direction the nose of the quad is pointing in. More precisely, we want the angle in degrees between the nose of the quad and "true North". When set up correctly, the returned value should, for example, be 90 degrees when the nose of the quad is pointing due East, 180 degrees when pointing due South, etc.
 
-Mag information is essential for position hold, which we intend to support.
+Mag information is essential for position hold, which we intend to support, and improves behaviour in GPS Rescue.
 
-GPS Rescue does not require Mag data, but heading control may be improved if reliable, accurate Mag information is available, especially during descents on windy days. Note: DO NOT enable Mag if you rely on GPS rescue, unless you are ABSOLUTELY CERTAIN that the Mag data is accurate and reliable, and have confirmed that enabling Mag makes a significant improvement in rescue behaviour!
+GPS Rescue will have improved eading control if reliable, accurate Mag information is available, especially during ascents and descents on windy days.
+
+:::note
```

Comment:
```suggestion
:::warning
```

Reply:
@VitroidFPV do you think this works with .MD anyway?

Reply:
I think it should compile this regardless of MD or MDX, I will quickly check